packages/curve-common/src/widgets/Header/types.ts

```diff
@@ -35,6 +36,8 @@ export type BaseHeaderProps<TChainId = number> = {
   isLite?: boolean
   ChainProps: ChainSwitcherProps<TChainId>
   WalletProps: ConnectWalletIndicatorProps
+  BannerProps: GlobalBannerProps
+  bannerHeight: number | undefined
```

Comment:
Couldn't this be part of BannerProps? I know it's for cypress testing, but it's still banner related.

Reply:
this is to adjust the mobile drawer and the desktop header so they go under the banner. It's not passed to the banner, but it could be part of the object